tests/translator/input/api_with_custom_base_path.yaml

```diff
@@ -32,3 +32,18 @@ Resources:
           HostedZoneId:
             Ref: HostedZoneId
           IpV6: true
+
+  MyApiWithoutNormalizedBasePath:
+    Type: AWS::Serverless::Api
+    Properties:
+      OpenApiVersion: 3.0.1
+      StageName: Prod
+      Domain:
+        DomainName: MyEdgeDomainName
+        CertificateArn: MyEdgeDomainCert
+        EndpointConfiguration: EDGE
+        NormalizeBasePath: false
```

Comment:
Would be clearer if it used `BasePath` with slashes as well, to make sure we don't revert this later and tests keep passing.